test/e2e/mock-e2e.js

```diff
@@ -188,6 +188,17 @@ async function setupMocking(
       };
     });
 
+  await server
+    .forGet(
+      'https://subscription.dev-api.cx.metamask.io/v1/subscriptions/eligibility',
+    )
+    .thenCallback(() => {
+      return {
+        statusCode: 200,
+        json: [],
+      };
+    });
+
```

Comment:
this is needed to avoid console error which makes spec fails. Now the onboarding tests make calls to this API (bc of the new fixture), so we return this as a global mock, to avoid the error.